GTG/core/urlregex.py

```diff
@@ -28,7 +28,7 @@
 UTF_CHARS = r'a-z0-9_\u00c0-\u00d6\u00d8-\u00f6\u00f8-\u00ff'
 
 SUBST_DICT = {
-    "domain": r'([\.-]|[^\s_\!\.\/])+\.[a-z]{2,}(?::[0-9]+)?',
+    "domain": r'([\.-]|[^\s_\!\.\/])+\.[a-z]{1,}(?::[0-9]+)?',
```

Comment:
```suggestion
    "domain": r'([\.-]|[^\s_\!\.\/])+\.[a-z]+(?::[0-9]+)?',
```
I can't double-check right now, but it should be synonymous?